check-web.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
should this file be in the root folder? I've been putting all health checks in /assets/utils - seems like a better standard.

Reply:
I was thinking about putting this file in the `./scripts/` folder cause `./assets` no longer exists. Take [a look](https://github.com/Start9Labs/hello-world-wrapper/tree/update/0.3.1).